src/shared/util.js

```diff
@@ -288,27 +288,9 @@ function deprecated(details) {
   console.log('Deprecated API usage: ' + details);
 }
 
-// Fatal errors that should trigger the fallback UI and halt execution by
-// throwing an exception.
-function error(msg) {
-  if (verbosity >= VERBOSITY_LEVELS.errors) {
```

Comment:
we will keep that for compatibility, also we may add error() back, but without throwing for recording critical stuff (e.g. recovery that we often hide under warn())